src/watertap_contrib/seto/analysis/net_metering/pysam_data/utilityrate5.json

```diff
@@ -0,0 +1,4 @@
+{
+	"en_electricity_rates" : 0,
+	"degradation" : [ 0 ]
```

Comment:
Why do we need this file?

Reply:
I don't know exactly. This is a question for @Matthew-Boyd.

Reply:
This is one of the four files that contain PySAM model parameters, each for their respective submodels. Organizing the parameters in files this way is also how they are exported from the SAM UI and is thus the easiest way to transfer parameters to PySAM from a SAM project.